newIDE/app/src/VariablesList/index.js

```diff
@@ -254,43 +252,78 @@ export default class VariablesList extends React.Component<Props, State> {
     );
   }
 
-  render() {
+  onAddVariable = (variableName = false) => {
     const { variablesContainer } = this.props;
+    const variable = new gd.Variable();
+    variable.setString('');
```

Comment:
A Variable already contains an empty string by default.

Reply:
I copied it from
https://github.com/4ian/GDevelop/pull/908/files#diff-c81b7be87704fa6d79ea7db7b482ce1cL285